### DISCUSSION TITLE
Sync complete button - gets a new behaviour

### DISCUSSION THREAD

**Discussion by jencastrodoesstuff** [digidem/mapeo-desktop#589] [Ideas]
**Describe the idea**
After the sync is complete the sync button says complete. EVERYONE clicks it hoping for an affirming response. What if they got one?!

A little status report on sync with that device.
Device x sent 43 new observations, 3 edited observations, 1 deleted.
You sent device x 16 new observations, 29 edited observations, 10 deleted.
BONUS!
Editing conflict Observation [category], [date], was edited by both devices. Edits by [you/device x]  were most recent and have been saved.  

**What's your role? Where and how do you currently use the application?**
Jen

**What is your ideal step by step workflow?**
click the complete button. 
see sync report
click x to close
see device with "last synced" info

**Have you thought about what it might look like?**
a blue box with white text over the device white box.

**Additional context**

**Comment by okdistribute** [comment]
Yes!
Right now we have no way in Mapeo Core to definitively know if a peer if we are syncronized with is up to date, and what differences there are from the last sync until now, but in theory this is possible and /most/ of the code is there, hidden in the filesystem/hypercore. It'll be a bit easier after the new protocol work is done as well.
